lib/gtfsplus/containers/ActiveGtfsPlusEditor.js

```diff
@@ -43,12 +43,15 @@ const mapStateToProps = (state: AppState, ownProps: Props) => {
   const feedSource = project && project.feedSources
     ? project.feedSources.find(fs => fs.id === feedSourceId)
     : null
-
+  const feedVersion = feedSource && feedSource.feedVersions
```

Comment:
Could a selector be used to get this data?

Reply:
I'm not sure. We would need `feedVersionId` in the selector, but I don't know if there is a great way to do that.